Dockerfile

```diff
@@ -1,24 +1,28 @@
 FROM node:14-alpine
 
-# Install dockerize so that we can wait for hubs to be up
-RUN apk update && apk add wget
+RUN apk update && apk upgrade
 
+# Install dockerize so that we can wait for hubs to be up
+RUN apk add wget
 ENV DOCKERIZE_VERSION v0.6.1
-
 RUN wget https://github.com/jwilder/dockerize/releases/download/$DOCKERIZE_VERSION/dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz \
   && tar -C /usr/local/bin -xzvf dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz \
   && rm dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz
 
-# Standard Ruby Packages
-RUN apk upgrade && apk add --upgrade ruby
+# Ruby Packages
+RUN apk add --upgrade ruby ruby-dev ruby-json ruby-etc
 
+# nokogiri compilation (C-gem)
+RUN apk add gcc libc-dev libxslt-dev libxml2-dev zlib-dev build-base
+
+# git for repository-based gems
+RUN apk add --no-cache git
```

Comment:
Included git as `citizens-advice-style` gem is installed via git. Not published to rubygems (maybe it should be?)

Reply:
This is something I remember "briefly" raising. If not rubygems an internal forge. I think this is a very good point to make. I'll message you offline and find out who to speak to next for this.